api/grant.go

```diff
@@ -146,6 +146,7 @@ func (r ListGrantsRequest) SetPage(page int) Paginatable {
 type GrantRequest struct {
 	User      uid.ID `json:"user"`
 	Group     uid.ID `json:"group"`
+	Name      string `json:"name"`
```

Comment:
```suggestion
	UserName      string `json:"userName"`
```

Just specifying `name` looks like naming the grant to me.

Reply:
Ya, this needs to be `UserName`, because groups also have a name, so even `SubjectName` is ambiguous.

Reply:
It seems odd this references user by id or name but group only by id. Does it also need `GroupName`?